docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md

```diff
@@ -48,23 +48,23 @@ function universalReceiverDelegate(
 ) public payable returns (bytes memory result)
 ```
 
-Writes the keys of the received **LSP7DigitalAsset**, **LSP8IdentifiableDigitalAsset** assets and **LSP9Vault** vaults into the account storage according to the **LSP5ReceivedAssets** and **LSP10ReceivedVaults** Standard.
+The function writes the keys of the received **LSP7DigitalAsset**, **LSP8IdentifiableDigitalAsset**, and **LSP9Vault** contract addresses into the account storage according to the **LSP5ReceivedAssets** and **LSP10ReceivedVaults** Standard.
 
-The keys representing an **asset** or a **vault** are cleared when the asset/vault is no longer owned by the account.
+The keys representing an asset/vault are cleared when the asset/vault is no longer owned by the account.
 
 #### Parameters:
 
-| Name     | Type    | Description                                                          |
-| :------- | :------ | :------------------------------------------------------------------- |
-| `sender` | address | The token/vault smart contract address.                              |
-| `typeId` | bytes32 | The token hooks.                                                     |
-| `data`   | bytes   | The data associated with the asset or vault transfer (concatenated). |
+| Name     | Type    | Description                                                                  |
+| :------- | :------ | :--------------------------------------------------------------------------- |
+| `sender` | address | The token's or vault's smart contract address.                               |
+| `typeId` | bytes32 | The token's hooks that are connected the the contract.                       |
```

Comment:
I think this should be reverted, as hooks are not connected to contracts, but it's more something independant.
@YamenMerhi what do you think?

Reply:
reverted to "The token hooks of the contract. "